bin/jetty_wait

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env sh
+
+until $(curl --output /dev/null --silent --head --fail http://localhost:8985/solr); do
+    printf '.'
+    sleep 1
+done
```

Comment:
While not strictly necessary here, it's a best practice to add a final newline to source files. Rubocop flags those up for us in ruby, but not in shell scripts like this.

Part of the POSIX standard:
http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206

Reply:
I will remove this file since I removed it in my previous commit from the config.yml.